src/app/component/user/components/profile/profile-service/profile.service.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { Language } from '@language-service/Language';
+import { LanguageService } from '@language-service/language.service';
+import { BehaviorSubject } from 'rxjs';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { ProfileService } from './profile.service';
+
+
+describe('ProfileService', () => {
+
+  let profileService: ProfileService;
+  let httpMock: HttpTestingController;
+
+  let localStorageServiceMock: LocalStorageService;
+  localStorageServiceMock = jasmine.createSpyObj('LocalStorageService', ['userIdBehaviourSubject']);
+  localStorageServiceMock.userIdBehaviourSubject = new BehaviorSubject(1111);
+
+  let languageServiceMock: LanguageService;
+  languageServiceMock = jasmine.createSpyObj('LanguageService', ['getCurrentLanguage']);
+  languageServiceMock.getCurrentLanguage = () => 'en' as Language;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [ HttpClientTestingModule ],
+      providers: [
+        ProfileService,
+        { provide: LocalStorageService, useValue : localStorageServiceMock },
+        { provide: LanguageService, useValue : languageServiceMock }
+      ]
+    });
+    profileService = TestBed.get(ProfileService);
+    httpMock = TestBed.get(HttpTestingController);
+  });
+
+  afterEach(() => {
+    httpMock.verify();
+  });
+
+  it('should be created', () => {
+    const service: ProfileService = TestBed.get(ProfileService);
+    expect(service).toBeTruthy();
+  });
+
+  it('should set user id', () => {
+    let userId = null;
+    // @ts-ignore
+    profileService.localStorageService.userIdBehaviourSubject
+    .subscribe(id => userId = id);
+    expect(userId).toBe(1111);
+  });
+
+  describe('getFactsOfTheDay', () => {
+    it('should return fact of the day', () => {
+      const fact = { id: 1, content: 'Great day!'};
+      profileService.getFactsOfTheDay().subscribe(info => {
+        expect(info.content).toBe('Great day!');
+      });
+
+      const req = httpMock.expectOne(`https://greencity.azurewebsites.net/factoftheday/?lang=en`);
+      expect(req.request.method).toBe('GET');
+      req.flush(fact);
+    });
+  });
+
+  describe('getUserInfo', () => {
+    it('should return user info', () => {
+      const userInfo = {
+        city: 'Lviv',
+        firstName: 'Anton',
+        userCredo: 'Eco',
+        profilePicturePath: 'somepath',
+        rating: 1999,
+        showEcoPlace: true,
+        showLocation: false,
+        showShoppingList: true,
+        socialNetworks: []
+      };
+
+      profileService.getUserInfo().subscribe(info => {
+        expect(info.rating).toBe(1999);
+      });
+
+      const req = httpMock.expectOne(`https://greencity-user.azurewebsites.net/user/1111/profile/`);
```

Comment:
https://greencity-user.azurewebsites.net/user/ -> make constant with this at line 11